testing-guidance/workflow-tests/scripting/script-python-tests.md

```diff
@@ -0,0 +1,37 @@
+# Python Scripts Workflow Tests
+
+Testing in this area should focus on the functionality of executing python scripts.
+
+## Common Issues to Watch For
+
+Test guidance will sometimes note specific issues to watch for. The common issues below should be watched for through all testing, even if unrelated to the current workflow being tested.
+1. Asset processor errors when importing/executing python scripts
+2. Warnings or errors that appear in the Editor console
+3. Unresponsive UI elements 
+
+### Platforms:
+- Windows
+- Linux
+
+### Documents and Common Terms
+[O3DE Documentation: Editor Python Bindings Gem](https://www.o3de.org/docs/user-guide/gems/reference/script/python/editor-python-bindings/) <br>
+
+
+<b>Python Scripts</b>: Tool that lists python script samples and allows to execute those scripts.
+
+<b></b>
```

Comment:
Remove

Reply:
Removed empty bold braces, should line 20 also be removed as it is highlighted?

Reply:
No, that's github displaying nearby lines when a single line is selected. It's a feature I'm not happy with due to this exact reason.